src/main/java/uk/gov/justice/hmpps/prison/api/resource/AppointmentsResource.java

```diff
@@ -53,10 +55,10 @@ public List<CreatedAppointmentDetails> createAppointments(
         @ApiResponse(responseCode = "404", description = "The appointment was not found"),
         @ApiResponse(responseCode = "403", description = "The client is not authorised for this operation")
     })
-    @Operation(summary = "Delete an appointment.", description = "Delete appointment.")
+    @Operation(summary = "Delete an appointment.", description = "Requires role GLOBAL_APPOINTMENT and write scope")
     @ResponseStatus(HttpStatus.NO_CONTENT)
+    @PreAuthorize("hasRole('GLOBAL_APPOINTMENT') and hasAuthority('SCOPE_write')")
```

Comment:
This PR involves moving annotations from the service layer (no behaviour changes hopefully!)